tests/func/test_checkout.py

```diff
@@ -422,31 +421,29 @@ def test(self):
         write_calls = self.filter_out_empty_write_calls(write_calls)
         self.write_args = [w_c[1][0] for w_c in write_calls]
 
-        pattern = re.compile(".*\\[.{30}\\].*%.*")
-        progress_bars = [
-            arg
-            for arg in self.write_args
-            if pattern.match(arg) and "unpacked" not in arg
-        ]
-
+        pattern = re.compile(r"%\|\W+\| .*\[.*\]")
```

Comment:
a lot of this shouldn't really be required thanks to https://github.com/tqdm/tqdm/tree/master/tqdm/tests

Reply:
@casperdcl please feel free to remove tests that you think are no longer needed.

Reply:
well I've already fixed/updated this test so might as well leave it in :)

Reply:
removed